src/lib/compute.ts

```diff
@@ -21,7 +21,7 @@ const mergeDefaults = (
 ) => {
   if (inputs) {
     const response = defaults
-      ? inputs.map(input => mergeObjects(input, defaults))
+      ? inputs.map(input => mergeObjects({...defaults}, input))
```

Comment:
Hi @alexzurbonsen , thanks for your contribution. 
Just wandering if there is specific reason for using object destructuring on defaults?

Reply:
Hi @MariamKhalatova, that is because the function uses lodash's `mergeWith` function. And that writes `input` into `defaults`. That is why we need to copy the top level of `defaults`, I think.

Reply:
The docs: [mergeWith](https://lodash.com/docs/4.17.15#mergeWith) and [merge](https://lodash.com/docs/4.17.15#merge)

Reply:
I added a test, that might help understand the issue. If you replace `{...defaults}` with `defaults` and run the test, you can see how the object reference kicks in.

I add to export `mergeDefaults` to make this testable. But I guess its worth it.